templates/README.md

```diff
@@ -20,15 +20,22 @@ clusterctl generate cluster ibm-vpc-1 --kubernetes-version v1.14.3 \
 
 ## Power VS
 
+> **Note:**
```

Comment:
would be better to add this in the start as it is common for both VPC and PowerVS